osu.Game/Overlays/Comments/CommentsContainer.cs

```diff
@@ -336,5 +409,40 @@ private void load()
                 });
             }
         }
+
+        private partial class NewCommentEditor : CommentEditor
+        {
+            [Resolved]
+            public Bindable<CommentableType> CommentableType { get; set; }
+
+            [Resolved(name: cid)]
+            public BindableLong CommentableId { get; set; }
+
+            protected override LocalisableString FooterText => default;
```

Comment:
Is there a reason why this is empty? Looks perfectly implementable to me.

Reply:
Web has `Press enter to post. Use shift+enter for new line.` here. It can't be used because the box isn't multiline. What string should put?

Reply:
Hmm, fair point. Maybe leave a TODO at least then.